samples/migrations/requirements.txt

```diff
@@ -1,13 +1,13 @@
-langchain-chroma==0.2.2
-langchain-core==0.3.41
-langchain-google-alloydb-pg==0.9.3
-langchain-milvus==0.1.8
-langchain-pinecone==0.2.3
+langchain-chroma==0.2.3
+langchain-core==0.3.55
+langchain-google-alloydb-pg==0.10.0
+langchain-milvus==0.1.9
+langchain-pinecone==0.2.5
 langchain-qdrant==0.2.0
 langchain-weaviate==0.0.4
 pinecone-client==6.0.0
-pymilvus==2.5.5
-pytest-asyncio==0.25.3
+pymilvus==2.5.6
```

Comment:
The latest pymilvus is `2.5.7` but we can only migrate till `2.5.6` because it seems pymilvus latest is not compatible with `langchain-milvus`.